metadata/src/test/java/edu/unc/lib/dl/util/DepositStatusFactoryIT.java

```diff
@@ -0,0 +1,143 @@
+/**
+ * Copyright 2016 The University of North Carolina at Chapel Hill
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package edu.unc.lib.dl.util;
+
+import static org.junit.Assert.*;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.UUID;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+
+import edu.unc.lib.dl.util.RedisWorkerConstants.DepositAction;
+import edu.unc.lib.dl.util.RedisWorkerConstants.DepositField;
+import edu.unc.lib.dl.util.RedisWorkerConstants.DepositState;
+import redis.clients.jedis.JedisPool;
+
+@RunWith(SpringJUnit4ClassRunner.class)
+@ContextConfiguration({"/spring-test/cdr-client-container.xml"})
+public class DepositStatusFactoryIT {
+	
+	private DepositStatusFactory factory;
+	@Autowired
+	private JedisPool jedisPool;
+	
+	@Before
+	public void init() {
+		factory = new DepositStatusFactory();
+		factory.setJedisPool(jedisPool);
+		jedisPool.getResource().flushAll();
```

Comment:
I know this is a test, but we should still make sure to release the jedis resource when we're done with it